ledger/eval/eval.go

```diff
@@ -1511,7 +1802,17 @@ func (eval *BlockEvaluator) GenerateBlock() (*ledgercore.ValidatedBlock, error)
 		return nil, err
 	}
 
-	vb := ledgercore.MakeValidatedBlock(eval.block, eval.state.deltas())
+	// look up set of participation accounts passed to GenerateBlock (possible proposers)
+	finalAccounts := make(map[basics.Address]ledgercore.AccountData, len(addrs))
```

Comment:
from what I see `UnfinishedBlock. finalAccounts` are only used in `ContainsAddress` that does not need account state so that this look can be removed.

Reply:
I think your review came in just before I pushed.
https://github.com/jannotti/go-algorand/blob/absenteeism/ledger/ledgercore/validatedBlock.go#L88

Reply:
see FinishBlock now, thanks.